<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QLabel" name="leftwindow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/rightwindow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rightwindow">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>0</y>
     <width>200</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/rightwindow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="he">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mon1l">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mon2l">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mon3l">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mon4l">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topwindow">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>0</y>
     <width>601</width>
     <height>104</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiU-ExtB</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/top2.jpg);</string>
   </property>
   <property name="text">
    <string>  MAGIC TOWER : SEER CREED</string>
   </property>
  </widget>
  <widget class="QLabel" name="skillwindow">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>700</y>
     <width>600</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bossl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>150</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wall1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wall2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>40</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wall3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wall4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>40</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wall5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wall6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>380</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wall7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>40</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wall8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>40</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wall9">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="waterl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>680</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="shieldl">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="swordl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>610</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="winl">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>300</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="firstskill">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>700</y>
     <width>150</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/sec.jpg);</string>
   </property>
   <property name="text">
    <string> 无能狂怒
(电闪雷鸣)</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondskill">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>700</y>
     <width>150</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/skill1.jpg);</string>
   </property>
   <property name="text">
    <string>杨永信电疗
(电流爆炸)</string>
   </property>
  </widget>
  <widget class="QLabel" name="thirdskill">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>700</y>
     <width>150</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/third.jpg);</string>
   </property>
   <property name="text">
    <string>    浪
(瀑流千击)</string>
   </property>
  </widget>
  <widget class="QLabel" name="forthskill">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>700</y>
     <width>150</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/forth.jpg);</string>
   </property>
   <property name="text">
    <string>   冥狗
(煌炎烈刃)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/attack.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="health">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hea">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/resources/pic/hea.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>870</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>憨憨制作组</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>960</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>V 8.0.1 beta</string>
   </property>
  </widget>
  <widget class="QLabel" name="wall10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>104</y>
     <width>599</width>
     <height>596</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/back3.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>811</x>
     <y>624</y>
     <width>171</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>制作组成员：

   高文畅
   刘方圆
   王昊天
   刘智</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/wall.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>501</y>
     <width>161</width>
     <height>194</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/pic/logo.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>leftwindow</zorder>
  <zorder>rightwindow</zorder>
  <zorder>he</zorder>
  <zorder>mon1l</zorder>
  <zorder>mon2l</zorder>
  <zorder>mon3l</zorder>
  <zorder>mon4l</zorder>
  <zorder>topwindow</zorder>
  <zorder>skillwindow</zorder>
  <zorder>bossl</zorder>
  <zorder>wall1</zorder>
  <zorder>wall2</zorder>
  <zorder>wall3</zorder>
  <zorder>wall4</zorder>
  <zorder>wall5</zorder>
  <zorder>wall6</zorder>
  <zorder>wall7</zorder>
  <zorder>wall8</zorder>
  <zorder>wall9</zorder>
  <zorder>waterl</zorder>
  <zorder>shieldl</zorder>
  <zorder>swordl</zorder>
  <zorder>winl</zorder>
  <zorder>firstskill</zorder>
  <zorder>secondskill</zorder>
  <zorder>thirdskill</zorder>
  <zorder>forthskill</zorder>
  <zorder>label</zorder>
  <zorder>health</zorder>
  <zorder>hea</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>wall10</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
